sgt-back-end/index.js

```diff
@@ -0,0 +1,157 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+    from "grades"
+  `;
+
+  db.query(sql)
+    .then(
+      result => {
+        res.json(result.rows);
+      }, err => {
+        console.error(err);
+        res.status(500).send('Error: an unexpected error has occurred');
```

Comment:
Ok, great job overall, but you should make sure that your Application Programming **Interface** is consistent and always responds with JSON. `{ error: 'an unexpected error occurred' }`.